app/src/main/java/com/instacart/android/challenges/MainActivityViewModel.kt

```diff
@@ -1,15 +1,11 @@
 package com.instacart.android.challenges
 
-import io.reactivex.annotations.Nullable
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
 
-class MainActivityViewModel {
+class MainActivityViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
I'm not strongly opposed to it, but curious why we need it?

Reply:
^ Chatted via Slack.